packages/nextjs/src/common/wrapApiHandlerWithSentryVercelCrons.ts

```diff
@@ -11,18 +11,17 @@ type EdgeRequest = {
 /**
  * Wraps a function with Sentry crons instrumentation by automaticaly sending check-ins for the given Vercel crons config.
  */
-export function wrapApiHandlerWithSentryVercelCrons<F extends (...args: any[]) => any>(
-  handler: F,
-  vercelCronsConfig: VercelCronsConfig,
-): F {
+export function wrapApiHandlerWithSentryVercelCrons<
+  F extends (args: [NextApiRequest | EdgeRequest | undefined] | undefined) => any,
```

Comment:
m: isn't this technically breaking?

Reply:
hmm, true I guess... 🤔 I guess I'll have to make it "less correct" inside of the proxy instead - I'll try it out.

Reply:
reverted the signature here!